python/hsml/sklearn/model.py

```diff
@@ -0,0 +1,66 @@
+#
+#   Copyright 2021 Logical Clocks AB
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from hsml.model import Model
+import humps
+
+
+class Model(Model):
```

Comment:
How are these model and signature classes under the different hsml directories different?

Reply:
What I'm achieving with this structure is that when you invoke `ModelRegistry.tensorflow.create_model(...)` it instantiates the model class from `hsml.tensorflow.model.Model`. Which is why there is a bunch of different Model classes (one for tensorflow, torch, sklearn, python). There is a great deal of code duplication here for setting the attributes like metrics, name etc. I need to figure out a way to reduce/remove that, but I want to do that in the next iteration. The only main difference is that we set the framework="TENSORFLOW" here to set the model framework.

As for signature, also fair bit of code duplication. I wanted to achieve tensorflow.create_model(...), sklearn.create_model(...) etc... which works for now, and I have an idea how to refactor it after we get this in.